templates/terraform/constants/tpu_node.erb

```diff
@@ -12,6 +12,13 @@ func compareTpuNodeSchedulingConfig(k, old, new string, d *schema.ResourceData)
 	return false
 }
 
+
+// The API takes the project_id in the network field and returns it as a number which causes permadiff.
```

Comment:
Is this a bug unrelated to your change? We might want to file a bug report for this.

Reply:
It is an upstream bug. https://issuetracker.google.com/issues/167838005